core-bundle/assets/controllers/dropdown-controller.js

```diff
@@ -0,0 +1,97 @@
+import { Controller } from '@hotwired/stimulus';
+
+export default class DropdownController extends Controller {
+    static targets = ['button', 'dropdown'];
```

Comment:
maybe this should be `menu` or `control` (as in aria) target and not `dropdown`, since we use it for the main menu as well? Ideally, the controller could toggle anything, no?

Reply:
Renamed
- `buttonTarget` to `controllerTarget` eea1f45130da92434ebddd6d43e401d0d04b79b7
- `dropdownTarget` to `controlsTarget` eea1f45130da92434ebddd6d43e401d0d04b79b7
- `dropdown-controller` to `toggle-state-controller` a33dbf058836a415def2317f6f4548052cb15254